run.py

```diff
@@ -1080,6 +1100,11 @@ def setting_post(
         if allow_origin is None:
             allow_origin = ""
 
+        if user_dictionary_file.filename:
+            import_user_dict(
+                parse_dict(user_dictionary_file.file), override=allow_override
+            )
+
```

Comment:
もしこちらのコメントの通りに変更する場合はこんな感じのコードになると思います。
（もし不正なファイルだった場合、500エラーになってしまうため、いい感じにエラーハンドリングすると良さそうです）
https://github.com/VOICEVOX/voicevox_engine/pull/676/files#r1181180902
```suggestion
        if user_dictionary_file.filename:
            import_user_dict(
                {k: UserDictWord(**v) for k,v in json.load(user_dictionary_file.file).items()},
                override=allow_override
            )

```